test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -62,6 +62,7 @@
     ("aten::grid_sampler_2d_backward", datetime.date(2021, 10, 21)),
     ("prim::TensorExprDynamicGuard", datetime.date(2021, 11, 20)),
     ("caffe2::", datetime.date(2021, 10, 23)),
+    ("aten::hardtanh", datetime.date(2021, 11, 1)),
```

Comment:
btw the risk of adding an optimistic date here is that if this merges to master within 24 hours after the date specified, the BC test will start failing on master (for a day, before it stops).

I usually pessimistically put something ~2-3 weeks in the future, maybe December 1st